### PR TITLE
Added support for building with Xcode 10.1 / iOS 12.1 SDK

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

**Comment by livecodepanos** [comment]
WIP until we have prebuilts for iOS 12.1

File: docs/notes-base/platforms.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ It will run on the following versions of Android:
 * 6.0 (Marshmallow)
 * 7.x (Nougat)
 * 8.x (Oreo)
+* 9.x (Pie)
```
Comment (livecodeali) [inline docs/notes-base/platforms.md]: This change should probably be in a separate PR, or at least a separate commit

Comment (livecodepanos) [reply docs/notes-base/platforms.md]: Sure, I'll submit a separate PR.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 60db038

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 60db038322dcdabece534fb4cd1bc2279766388e

**Comment by livecode-vulcan** [comment]
😎 test success 60db038322dcdabece534fb4cd1bc2279766388e

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/258)
* try-community-armv7-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-sdk26_ndk16r15/builds/250)
* try-community-arm64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-sdk26_ndk16r15/builds/250)
* try-community-x86-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-sdk26_ndk16r15/builds/250)
* try-community-x86_64-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-sdk26_ndk16r15/builds/250)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1772)
* try-community-universal-ios-iphoneos12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.1/builds/0)
* try-community-universal-ios-iphonesimulator12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.1/builds/0)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1803)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1793)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1792)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1827)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1767)
